### PR TITLE
Update README about build image

### PR DESCRIPTION
**Pull request by bytemain**
Change the plugin name and sync content from Dockerfile-alpine

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -567,18 +567,27 @@ If you need additional Caddy plugins, or need to use a specific version of Caddy
...
     --with <additional-plugins>
 
 FROM caddy:${CADDY_VERSION}-alpine
 
+EXPOSE 80 443 2019
+
+ENV XDG_CONFIG_HOME /config
+ENV XDG_DATA_HOME /data
+
+RUN apk add -U --no-cache ca-certificates curl
```
Comment (francislavoie) [inline README.md:584]: These are all unnecessary/redundant, the base image already sets these appropriately.

File: README.md [hunk 2]
```diff
@@ -567,18 +567,27 @@ If you need additional Caddy plugins, or need to use a specific version of Caddy
...
+
 COPY --from=builder /usr/bin/caddy /usr/bin/caddy
 
-CMD ["caddy", "docker-proxy"]
+ENTRYPOINT ["/bin/caddy"]
+
+CMD ["docker-proxy"]
```
Comment (francislavoie) [inline README.md:590]: I don't agree with this change, this makes it harder to get a shell into the container with `/bin/sh` to muck around and debug when necessary.

Comment (bytemain) [reply README.md]: Thanks for your advice, I agree with that.

by the way, these configs is copied from the `Dockerfile-alpine`: https://github.com/lucaslorentz/caddy-docker-proxy/blob/master/Dockerfile-alpine#L13

**Comment by lucaslorentz** [comment]
Thank you @bytemain

**Pull request merged**